### PR TITLE
build-locally.py: Use Python 3 if multiple versions are available

### PR DESCRIPTION
**Pull request by henryiii**
This requires Python 3.6+, so it should list python3 as the runner.

Note that [PEP 394](https://www.python.org/dev/peps/pep-0394/) still recommends distributions provide `python -> python2`, and distributions like RedHat 8 are not providing an unversioned `python` command at all.

### CODE REVIEW

**Comment by scopatz** [comment]
Can you please provide a news entry: https://regro.github.io/rever-docs/news.html

**Comment by henryiii** [comment]
@scopatz I added an entry.

File: news/henryiii-build-locally-python3.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* `build-locally.py` now uses `python3` even if `python` is `python2` (Python 3.6+ was already required)
```
Comment (scopatz) [inline news/henryiii-build-locally-python3.rst]: This is rat, not me, so you need to use double backticks :(

Comment (henryiii) [reply news/henryiii-build-locally-python3.rst]: Ah, I see the file extension. Will correct tomorrow.

Comment (scopatz) [reply news/henryiii-build-locally-python3.rst]: Stupid autocorrect keyboard 😡

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**